edx/blockstore
# [SE-3451] Remove Python 3.5 support from docker/Makefile configuration.

## Description

This removes support for Python 3.5 from the Makefile and Docker configuration files.

## Author Comments, Concerns, and Open Questions

I've not fully reversed the changes I made to the Makefile originally, but only removed all handling of 3.5 specifically. This should make it easier to upgrade again later by providing a template to build from without having to support 3.5.

This PR is dependent on https://github.com/edx/blockstore/pull/89, since that PR removes support from CI for 3.5.

## Test Instructions

1. Shut down the blockstore if it's currently running.
2. Run `make easyserver`
3. Run the unit tests per the README.
4. Run the integration tests, also per the README.
5. Blow away your cached images for blockstore and the cached volumes. If you're feeling frisky, do `make dev.up` in devstack to make sure your other devstack containers are in active use, make sure blockstore is stopped, and run `docker system prune` to blow away everything else. If you've got other containers you care about, don't do that. Instead, the following may be enough:

```
docker volume ls |grep blockstore|awk '{print $2}'|xargs docker volume rm
```

This command will probably error out, giving you a list of volumes that can't be destroyed because they're attached to containers. `docker kill`, then `docker rm` each container hash it mentions, then run the above command again.

6. Run steps 2-4 again.

## Reviewers
